README.md

```diff
@@ -154,6 +158,55 @@ rrulestr('RRULE:FREQ=MONTHLY;COUNT=5;DTSTART=20120201T023000Z\nRDATE:20120701T02
 For more examples see
 [python-dateutil](http://labix.org/python-dateutil/) documentation.
 
+* * * * *
+
+### Timezone Support
+
+By default, `RRule` only correctly supports
+["floating" times or UTC timezones](https://tools.ietf.org/html/rfc2445#section-4.2.19).
+Optionally, it also supports use of the `TZID` parameter in the
+[RFC](https://tools.ietf.org/html/rfc2445#section-4.2.19)
+when the [Luxon](https://github.com/moment/luxon) library is provided. The 
+[specification](https://moment.github.io/luxon/docs/manual/zones.html#specifying-a-zone)
+and [support matrix](https://moment.github.io/luxon/docs/manual/matrix.html) for Luxon apply.
+
+Example with `TZID`:
+
+```js
+new RRule({
+  dtstart: new Date(Date.UTC(2018, 1, 1, 10, 30)),
+  count: 1,
+  tzid: 'Asia/Tokyo'
+}).all()
+
+// assuming the system timezone is set to America/Los_Angeles, you get:
+[ '2018-01-31T17:30:00.000Z' ]
+// which is the time in Los Angeles when it's 2018-02-01T10:30:00 in Tokyo.
+```
+
+Whether or not you use the `TZID` param, make sure to only use JS `Date` objects that are
+represented in UTC to avoid unexpected timezone offsets being applied, for example:
+
+```js
+// WRONG: Will produce dates with TZ offsets added
```

Comment:
Might be worth noting that this is how the library has worked up until now.  Changelog?

Reply:
Actually this is has been the case since 2.3.0, an update to the README has been overdue. Arguably, 2.3.0 was a breaking change although I thought of it at the time as a bugfix.